dali/core/dynlink_cufile.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <dlfcn.h>
+#include <stdio.h>
+#include <mutex>
+#include <string>
+#include <unordered_map>
+#include "dali/core/dynlink_cufile.h"
+
+namespace {
+
+typedef void *CUFILE;
```

Comment:
```suggestion
typedef void* CUFILE;
```
Not sure if we have any rules about it, but this reads better IMO

Reply:
Done